var/spack/repos/builtin/packages/mvdtool/package.py

```diff
@@ -24,12 +26,10 @@ class Mvdtool(CMakePackage):
     version('1.4', tag='v1.4')
 
     variant('mpi', default=True, description="Enable MPI backend")
-    variant('python', default=False, description="Enable Python bindings")
+    # variant('python', ...)  # Please use py-mvdtool instead
```

Comment:
remove ?

Reply:
Sure. Dont approve any PR if people try to reimplement this 😆